tests/standard_to_enhanced/planned_events_translator_test.py

```diff
@@ -326,23 +325,21 @@ def test_wzdx_creator(mock_dt, mock_dt_3, _):
 
     expected_wzdx = {
```

Comment:
Same here, should we pull into separate files?

Reply:
moved to planned_events_translator_data.py